jobs/transforms/user_audits_test.py

```diff
@@ -67,6 +67,40 @@ def test_process_reports_nothing_for_valid_uid(self):
         self.assert_pcoll_equal(output, [])
 
 
+class ValidateActivityMappingOnlyAllowedKeysTests(
+        job_test_utils.PipelinedTestBase):
+
+    NOW = datetime.datetime.utcnow()
+    USER_ID = 'test_id'
+    EMAIL_ID = 'a@a.com'
+    INCORRECT_KEY = 'audit'
+    ROLE = 'ADMIN'
+
+    def test_process_with_incorrect_keys(self):
+
+        test_model = user_models.PendingDeletionRequestModel(
+            id=self.USER_ID,
+            email=self.EMAIL_ID,
+            created_on=self.NOW,
+            last_updated=self.NOW,
+            role=self.ROLE,
+            pseudonymizable_entity_mappings={
+                models.NAMES.audit: {'key': 'value'}
```

Comment:
```suggestion
                models.NAMES.audit.value: {'key': 'value'}
```

Reply:
Also add a test for process with only correct keys.

Reply:
Done!